app/controllers/importer_controller.rb

```diff
@@ -657,4 +661,15 @@ def version_id_for_name!(project,name,add_versions)
     @version_id_by_name[name]
   end
 
+  def process_multivalue_custom_field(issue, custom_field, csv_val)
+    csv_val.split(',').map(&:strip).map do |val|
+      if custom_field.field_format == 'version'
+        version = Version.find_by_name val
+        version.id
+      else
+        val
```

Comment:
It's seems there are only 'version' and 'list' types. And yes, for 'list' types you just return an array of values straight up.